### ISSUE TITLE
vscode in macos dont generate new entries

### ISSUE THREAD

**Issue by jorge-teixeira** [pmahend1/resxpress#76]
Save the .resx dont regenerate the .cs file..
i have the generateStronglyTypedResourceClassOnSave : true

**Comment by pmahend1** [comment]
Thanks for reporting. Please share a reproducible example to investigate further.
Some tips:
- Have `dotnet` installed
- Make sure `resgen` is added in path.

If the above doesn't work then,
- Have Visual Studio for Mac installed 
- Open the project with resx file on Visual Studio for Mac as well.

**Comment by maxbrousseau** [comment]
I have the same issue. 

- I have dotnet installed
- resgen is added in path.

I am on MacOS.

**Comment by pmahend1** [comment]
Sorry for late response. Could you please provide sample to reproduce this? Please note `generateStronglyTypedResourceClassOnSave : true` support is limited. If you use Visual Studio IDE in conjunction, that should generate resource files on updates. I will see what I can do based on your reproducible example.

**Comment by pmahend1** [comment]
As I searched further I found that resgen.exe on Windows(.NET framework) is different than resgen of mono. Former one has option to generate strongly generated resource class file while latter does not.

https://social.msdn.microsoft.com/Forums/en-US/0caade81-822a-4997-a9c7-de854e008900/not-able-to-use-resgen-on-mac-to-generate-cs-strongly-types-resource-class-file?forum=xamarinforms

I also tried `dotnet build` `msbuild` commands on .csproj unsuccessfully. So at this point there is no working solution.
